### DISCUSSION TITLE
CycleGAN 9epoch result

### DISCUSSION THREAD

**Discussion by Dongbin-Lee-git** [bcaitech1/p4-fr-soccer#11] [Experiment]
CycleGAN으로 9에폭 돌았을떄 이렇게 나오네요...
- train은 15000장으로 했습니다.
- training parameter도 default에서 조금만 바꿔서 했는데 결과가 좋은데요....?
![image](https://user-images.githubusercontent.com/57521132/119574945-36421300-bdf1-11eb-82e9-50fb6e2cd66d.png)
![image](https://user-images.githubusercontent.com/57521132/119574960-3cd08a80-bdf1-11eb-9177-dba8c6e3558b.png)
- 형광펜도 잘 지워주네요 👍 
![image](https://user-images.githubusercontent.com/57521132/119575048-5e317680-bdf1-11eb-8edf-eec01145a64a.png)
![image](https://user-images.githubusercontent.com/57521132/119575051-612c6700-bdf1-11eb-9af9-ceb51130fbfc.png)
- 흐린애들은 아직 잘 안되네요 ( 살짝 흐린애들은 잘되요!! 이거는 엄청 흐린애라 ㅎㅎ.. )
![image](https://user-images.githubusercontent.com/57521132/119575205-9fc22180-bdf1-11eb-9622-3f6a826a4026.png)
![image](https://user-images.githubusercontent.com/57521132/119575211-a2bd1200-bdf1-11eb-8e95-176d85ab4229.png)

- 지금 모델이 모든 이미지를 512 size를 input으로 가지고 있어서 이 부분을 잡아주면 좀 괜찮을 거 같기도 하네요 ㅎㅎ..

**Comment by bellamy-k** [comment]
9에폭만에 결과가 괜찮네요 ㄷㄷ
